libr/parse/parse.c

```diff
@@ -468,6 +468,7 @@ static int filter(RParse *p, RFlag *f, char *data, char *str, int len, bool big_
 			case 80:
 				if (p && p->anal && p->anal->syscall) {
 					RSyscallItem *si;
+					//todo dont have core here, how to lookup?
 					si = r_syscall_get (p->anal->syscall, off, -1);
```

Comment:
Ditto. I have RParse here not core. Any ideas how to thread my way from p to get the imm value?

Reply:
why do you need RCore?

Reply:
You have RAnal via RAnalBind as well as a RCoreBind instance, so you have RCore in there too.

That's all you need to analyze that instruction, but anyway just focus on fixing the ones that you need, dont try to fix all of them at first try